v2/features/dill/RevenueStats.tsx

```diff
@@ -100,6 +118,21 @@ const RevenueStats: FC<Props> = ({ dill }) => {
                   />
                 </div>
               </div>
+              <div className="flex items-center font-bold text-sm">
+                <span className="text-foreground mr-1">
+                  {formatNumber(upcomingDistribution.weeklyEthAmount)}
+                </span>
+                <div className="inline-block w-6 h-6">
+                  <Image
+                    src="/ethereum.png"
```

Comment:
Please change to `/tokens/eth.png`. Going forward after the release of UI v2, the token names/labels will rely on the naming in pf-core's `ExternalTokenModel`, and the `public` assets will mainly fall into either `/tokens` or `/protocols`